### ISSUE TITLE
r2r tests failing: Method not found: 'Void Xunit.DelegatingXmlCreationSink..ctor(Xunit.IExecutionSink, System.Xml.Linq.XElement)'

### ISSUE THREAD

**Issue by jiangzeng** [dotnet/coreclr#20797]
**Jobs:**
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_gcstress15_tst/93/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_gcstress15_tst/93/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitminopts_tst/146/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitminopts_tst/146/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitforcerelocs_tst/149/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitforcerelocs_tst/149/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstress1_tst/146/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstress1_tst/146/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstress1_tiered_tst/30/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstress1_tiered_tst/30/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs0x1000_tst/145/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs0x1000_tst/145/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs0x10_tst/150/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs0x10_tst/150/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs0x80_tst/146/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs0x80_tst/146/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs1_tst/147/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs1_tst/147/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs2_tst/151/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs2_tst/151/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs3_tst/149/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs3_tst/149/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs4_tst/152/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs4_tst/152/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs8_tst/150/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_jitstressregs8_tst/150/)
[https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_no_tiered_compilation_tst/30/](https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_centos7.1_r2r_no_tiered_compilation_tst/30/)

**Log:**
```
EXEC : error : Method not found: 'Void Xunit.DelegatingXmlCreationSink..ctor(Xunit.IExecutionSink, System.Xml.Linq.XElement)'. [/mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/runtest.proj]
/mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/tests.targets(74,5): error MSB3073: The command "/mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/bin/tests/Linux.x64.Checked/Tests/Core_Root/corerun /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/bin/tests/Linux.x64.Checked/Tests/Core_Root/xunit.console.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/baseservices/compilerservices/baseservices.compilerservices.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/baseservices/exceptions/baseservices.exceptions.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/baseservices/multidimmarray/baseservices.multidimmarray.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/baseservices/threading/baseservices.threading.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/baseservices/varargs/baseservices.varargs.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/CoreMangLib/system/CoreMangLib.system.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Exceptions/Finalization/Exceptions.Finalization.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Exceptions/ForeignThread/Exceptions.ForeignThread.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/GC/API/GC.API.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/GC/Coverage/GC.Coverage.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/GC/Features/GC.Features.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/GC/LargeMemory/GC.LargeMemory.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/GC/Regressions/GC.Regressions.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/GC/Scenarios/GC.Scenarios.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/hosting/stress/hosting.stress.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/ArrayMarshalling/Interop.ArrayMarshalling.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/BestFitMapping/Interop.BestFitMapping.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/COM/Interop.COM.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/DllImportAttribute/Interop.DllImportAttribute.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/ExecInDefAppDom/Interop.ExecInDefAppDom.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/FuncPtrAsDelegateParam/Interop.FuncPtrAsDelegateParam.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/ICastable/Interop.ICastable.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/MarshalAPI/Interop.MarshalAPI.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/NativeCallable/Interop.NativeCallable.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/PInvoke/Interop.PInvoke.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/PrimitiveMarshalling/Interop.PrimitiveMarshalling.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/RefCharArray/Interop.RefCharArray.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/RefInt/Interop.RefInt.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/SimpleStruct/Interop.SimpleStruct.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/StringMarshalling/Interop.StringMarshalling.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/StructMarshalling/Interop.StructMarshalling.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Interop/StructPacking/Interop.StructPacking.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/BBT/JIT.BBT.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/CheckProjects/JIT.CheckProjects.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/CodeGenBringUpTests/JIT.CodeGenBringUpTests.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/Directed/JIT.Directed.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/Generics/JIT.Generics.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/HardwareIntrinsics/JIT.HardwareIntrinsics.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/IL_Conformance/JIT.IL_Conformance.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/Intrinsics/JIT.Intrinsics.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/jit64/JIT.jit64.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/Methodical/JIT.Methodical.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/opt/JIT.opt.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/Performance/JIT.Performance.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/Regression/JIT.Regression.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/RyuJIT/JIT.RyuJIT.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/JIT/SIMD/JIT.SIMD.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/binding/Loader.binding.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/classloader/Loader.classloader.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/lowlevel/Loader.lowlevel.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/multimodule/Loader.multimodule.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/NativeLibs/Loader.NativeLibs.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/regressions/Loader.regressions.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/Loader/versioning/Loader.versioning.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/readytorun/DynamicMethodGCStress/readytorun.DynamicMethodGCStress.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/readytorun/tests/readytorun.tests.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/reflection/DefaultInterfaceMethods/reflection.DefaultInterfaceMethods.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/reflection/GenericAttribute/reflection.GenericAttribute.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/reflection/ldtoken/reflection.ldtoken.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/reflection/regression/reflection.regression.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/reflection/Tier1Collectible/reflection.Tier1Collectible.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/tracing/eventactivityidcontrol/tracing.eventactivityidcontrol.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/tracing/eventlistener/tracing.eventlistener.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/tracing/eventsource/tracing.eventsource.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/tracing/runtimeeventsource/tracing.runtimeeventsource.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/tracing/tracecontrol/tracing.tracecontrol.XUnitWrapper.dll /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/../bin/tests/Linux.x64.Checked/tracing/tracevalidation/tracing.tracevalidation.XUnitWrapper.dll -parallel all -html /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/bin/Logs/TestRun.html -xml /mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/bin/Logs/TestRun.xml  -notrait category=outerloop -notrait category=failing -nocolor" exited with code 4. [/mnt/resource/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_centos7.1_r2r_gcstress15_tst/tests/runtest.proj]
    0 Warning(s)
    2 Error(s)
```

**Comment by BruceForstall** [comment]
This is not specific to CentOS or x64. I see it in arm64 Ubuntu16.04:

https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_cross_checked_ubuntu16.04_r2r_tst/7/consoleFull

and Ubuntu 14.04 arm32:

https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm_cross_checked_ubuntu_r2r_tst/34/consoleFull

and Windows x86:

https://ci.dot.net/job/dotnet_coreclr/job/master/view/x86/job/x86_checked_windows_nt_r2r/2370/consoleFull

and Windows x64:

https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/x64_checked_windows_nt_r2r/2631/consoleFull

**Comment by BruceForstall** [comment]
@sergiy-k Can you get someone to look at this?

The error happens if the framework is crossgen'ed before running tests.

**Comment by sergiy-k** [comment]
@janvorli, could you please take a look?

**Comment by janvorli** [comment]
@sergiy-k sure, I'll be happy to take a look.

**Comment by echesakovMSFT** [comment]
I believe that this failures started after I had merged a workaround for XUnit ( https://github.com/dotnet/coreclr/pull/20644). I confirmed locally that if I replace our custom xunit.console.dll back with its version from NuGet package r2r jobs won't fail anymore.

I also found a way to keep this workaround - we would need to replace xunit.runner.reporters.netcoreapp10.dll and xunit.runner.utility.netcoreapp10.dll that got built together with xunit.console.dll. I have tested this locally and I am testing this in CI right now (https://github.com/dotnet/coreclr/pull/20959).

I still don't know why this works with IL framework libraries and fails only with crossgened ones. @janvorli any idea?

/cc @RussKeldorph @BruceForstall
